microsetta_private_api/admin/tests/test_admin_repo.py

```diff
@@ -323,15 +324,21 @@ def make_tz_datetime(y, m, d):
             "barcode": test_barcode,
             "scan_timestamp": make_tz_datetime(2017, 7, 16),
             "sample_status": 'no-registered-account',
-            "technician_notes": "huh?"
+            "technician_notes": "huh?",
+            "observation_id": None,
+            "observations": None,
+            "category": None
```

Comment:
```suggestion
            "observations": None
```

Tests should reflect exactly what microsetta-admin is posting to microsetta-private-api.